purchase/purchases/tender/manage_blanket_orders.rst

```diff
@@ -0,0 +1,90 @@
+=========================================
+How to manage Blanket Purchase Agreements
+=========================================
+
+A **Blanket Purchase Order** (BPO), also known as Blanket Purchase Agreement or Call-Off Order, is a contract between you (the customer) and your supplier.  It is used to negotiate a discounted price.  The supplier is benefited by the economies of scale inherent in a large order.  You are benefited by being allowed to take multiple smaller deliveries over a period of time, at a lower price, without paying for the large order immediately.  Each small periodic delivery is called a release or call-off.
+
+As the BPO is a contract, it will have some prearranged conditions.  These usually include:
+
+-   Total quantity of each product to be delivered
+
+-   Completion deadline, by which you must take delivery of the total quantity
+
+-   Unit price for each product
+
+-   Delivery lead time in days for each release
+
+Activate the Purchase Tender function
+-------------------------------------
+
+The Blanket Purchase Order function is provided by the Purchase Tender function.  By default, the Purchase Tender is not activated. To be able to use BPOs,
+you must first activate the option.
+
+In the Purchases module, open the Configuration menu and click on
+Settings. In the Purchase Order section, locate the **Calls for
+Tenders** and tick the box Allow using call for tenders... (advanced),
+then click on **Apply**.
+
+.. image:: media/manage_multiple_offers00.png
+	:align: center
+
+Set Product Supplier and Lead Time
+----------------------------------
+
+Set up the supplier on the products that will be added to this blanket order.  Be sure to specify the prearranged **Delivery Lead Time**.
+
+For instructions, please read the chapter `How to set several suppliers on a product? <https://www.odoo.com/documentation/user/10.0/purchase/purchases/master/suppliers.html>`__
+
+Create a Blanket Purchase Order
+-------------------------------
+
+To create a new Blanket Purchase Order, open :menuselection:`Purchase --> Purchase Agreements
+(PA)`.
+
+.. image:: media/manage_multiple_offers01.png
```

Comment:
Did you forget to add the media?
Keep the format `manage_blanket_orders<number>.png`
Also don't forget to compress it using tools like [pngquant](https://pngquant.org/)